app/src/main/java/se/nullable/flickboard/model/layouts/FAMessagEase.kt

```diff
@@ -36,7 +36,10 @@ val FA_MESSAGEASE_MAIN_LAYER = Layer(
                     Direction.CENTER to Action.Text("م"),
                     Direction.TOP_RIGHT to Action.Text(
                         "\u200F", //RLM
-                        visualOverride = ActionVisual.Label("¶‹"),
+                        visualOverride = ActionVisual.Label(
+                            "¶‹",
+                            directionOverride = TextDirection.RightToLeft
```

Comment:
it should be  `TextDirection.LeftToRight` to be displayed as `¶‹`

Reply:
9bbb26780aa8752b8345d178de004660001c4390